gRPC-Core.podspec

```diff
@@ -35,7 +35,7 @@
 
 Pod::Spec.new do |s|
   s.name     = 'gRPC-Core'
-  version = '1.0.2'
+  version = '1.1.0-pre1'
```

Comment:
that is right... I am thinking of that too. Not only the version number but automation of the entire advancement process